diff --git a/dlls/winemac.drv/keyboard.c b/dlls/winemac.drv/keyboard.c
index a46d324..66cc4f2 100644
--- a/dlls/winemac.drv/keyboard.c
+++ b/dlls/winemac.drv/keyboard.c
@@ -238,14 +238,14 @@ static const struct {
     { VK_BACK,                  0x0E,           TRUE },     /* kVK_Delete */
     { 0,                        0,              FALSE },    /* 0x34 unused */
     { VK_ESCAPE,                0x01,           TRUE },     /* kVK_Escape */
-    { VK_RMENU,                 0x38 | 0x100,   TRUE },     /* kVK_RightCommand */
-    { VK_LMENU,                 0x38,           TRUE },     /* kVK_Command */
+    { 0,                        0,              FALSE },    /* kVK_RightCommand */
+    { 0,                        0,              FALSE },    /* kVK_Command */
     { VK_LSHIFT,                0x2A,           TRUE },     /* kVK_Shift */
     { VK_CAPITAL,               0x3A,           TRUE },     /* kVK_CapsLock */
-    { 0,                        0,              FALSE },    /* kVK_Option */
+    { VK_LMENU,                 0x38,           TRUE },     /* kVK_Option */
     { VK_LCONTROL,              0x1D,           TRUE },     /* kVK_Control */
     { VK_RSHIFT,                0x36,           TRUE },     /* kVK_RightShift */
-    { 0,                        0,              FALSE },    /* kVK_RightOption */
+    { VK_RMENU,                 0x38 | 0x100,   TRUE },     /* kVK_RightOption */
     { VK_RCONTROL,              0x1D | 0x100,   TRUE },     /* kVK_RightControl */
     { 0,                        0,              FALSE },    /* kVK_Function */
     { VK_F17,                   0x68,           TRUE },     /* kVK_F17 */
