diff --git a/loader/wine.inf.in b/loader/wine.inf.in
index edefbaac35e..d3a8e18f0ca 100644
--- a/loader/wine.inf.in
+++ b/loader/wine.inf.in
@@ -1,4 +1,4 @@
-;; .INF script for the basic Wine configuration
+﻿;; .INF script for the basic Wine configuration
 ;; Version: @PACKAGE_STRING@
 ;;
 ;; This should be run through setupapi:
@@ -37,6 +37,7 @@ AddReg=\
     ControlClass,\
     CurrentVersion,\
     Debugger,\
+    Replacements,\
     DirectX,\
     Fonts,\
     MCI,\
@@ -61,6 +62,7 @@ AddReg=\
     ControlClass,\
     CurrentVersion,\
     Debugger,\
+    Replacements,\
     DirectX,\
     Fonts,\
     MCI,\
@@ -88,6 +90,7 @@ AddReg=\
     CurrentVersion,\
     CurrentVersionWow64,\
     Debugger,\
+    Replacements,\
     DirectX,\
     Fonts,\
     MCI,\
@@ -137,6 +140,7 @@ AddReg=\
     CurrentVersion,\
     CurrentVersionWow64,\
     Debugger,\
+    Replacements,\
     DirectX,\
     MCI,\
     Misc,\
@@ -600,6 +604,15 @@ HKLM,%CurrentVersionNT%\AeDebug,"Auto",2,"1"
 HKCU,Software\Wine\Debug,"RelayExclude",2,"ntdll.RtlEnterCriticalSection;ntdll.RtlTryEnterCriticalSection;ntdll.RtlLeaveCriticalSection;kernel32.48;kernel32.49;kernel32.94;kernel32.95;kernel32.96;kernel32.97;kernel32.98;kernel32.TlsGetValue;kernel32.TlsSetValue;kernel32.FlsGetValue;kernel32.FlsSetValue;kernel32.SetLastError"
 HKCU,Software\Wine\Debug,"RelayFromExclude",2,"winex11.drv;winemac.drv;user32;gdi32;advapi32;kernel32"
 
+[Replacements]
+HKCU,Software\Wine\Fonts\Replacements,"MS UI Gothic"   ,,"VL PGothic"
+HKCU,Software\Wine\Fonts\Replacements,"Meiryo UI"      ,,"VL PGothic"
+HKCU,Software\Wine\Fonts\Replacements,"Yu Gothic UI"   ,,"VL PGothic"
+HKCU,Software\Wine\Fonts\Replacements,"ＭＳ ゴシック"  ,,"VL Gothic"
+HKCU,Software\Wine\Fonts\Replacements,"ＭＳ Ｐゴシック",,"VL PGothic"
+HKCU,Software\Wine\Fonts\Replacements,"ＭＳ 明朝"      ,,"ヒラギノ明朝 ProN W3"
+HKCU,Software\Wine\Fonts\Replacements,"ＭＳ Ｐ明朝"    ,,"ヒラギノ明朝 ProN W3"
+
 [DirectX]
 HKLM,Software\Microsoft\DirectX,"Version",,"4.09.00.0904"
 HKLM,Software\Microsoft\DirectX,"InstalledVersion",1,00,00,00,09,00,00,00,00
