diff --git a/dlls/winemac.drv/cocoa_window.m b/dlls/winemac.drv/cocoa_window.m
index 48d0651..9d112b0 100644
--- a/dlls/winemac.drv/cocoa_window.m
+++ b/dlls/winemac.drv/cocoa_window.m
@@ -1140,6 +1140,9 @@ - (void) orderBelow:(WineWindow*)prev orAbove:(WineWindow*)next activate:(BOOL)a
             [NSApp unhide:nil];
             wasVisible = [self isVisible];
 
+            [NSApp setPresentationOptions:NSApplicationPresentationAutoHideDock|
+                                          NSApplicationPresentationAutoHideMenuBar];
+
             if (activate)
                 [NSApp activateIgnoringOtherApps:YES];
 
