diff --git a/programs/iexplore/main.c b/programs/iexplore/main.c
index 73fec25..b622f3e 100644
--- a/programs/iexplore/main.c
+++ b/programs/iexplore/main.c
@@ -40,7 +40,7 @@ static BOOL check_native_ie(void)
     static const WCHAR wineW[] = {'W','i','n','e',0};
     static const WCHAR file_desc_strW[] =
         {'\\','S','t','r','i','n','g','F','i','l','e','I','n','f','o',
-         '\\','0','4','0','9','0','4','e','4',
+         '\\','0','4','1','1','0','3','a','4',
          '\\','F','i','l','e','D','e','s','c','r','i','p','t','i','o','n',0};
 
     size = GetFileVersionInfoSizeW(browseui_dllW, &handle);
