diff --git a/tools/Makefile.in b/tools/Makefile.in
index 3ee8473..19d2f6c 100644
--- a/tools/Makefile.in
+++ b/tools/Makefile.in
@@ -8,8 +8,6 @@ PROGRAMS = \
 	sfnt2fnt$(EXEEXT)
 
 MANPAGES = \
-	winemaker.de.UTF-8.man.in \
-	winemaker.fr.UTF-8.man.in \
 	winemaker.man.in
 
 C_SRCS = \
@@ -41,8 +39,6 @@ sfnt2fnt$(EXEEXT): sfnt2fnt.o
 .PHONY: install install-lib install-dev uninstall
 
 install install-lib::
-	$(INSTALL_DATA) $(srcdir)/wine.desktop $(DESTDIR)$(datadir)/applications/wine.desktop
-	-$(UPDATE_DESKTOP_DATABASE)
 
 install install-dev:: install-man-pages
 	$(INSTALL_SCRIPT) $(srcdir)/winemaker $(DESTDIR)$(bindir)/winemaker
