diff --git a/dlls/gdi32/freetype.c b/dlls/gdi32/freetype.c
index af4e9248f00..ab7abb31026 100644
--- a/dlls/gdi32/freetype.c
+++ b/dlls/gdi32/freetype.c
@@ -7033,13 +7033,13 @@ static FT_Int get_load_flags( UINT format )
     switch (format & ~GGO_GLYPH_INDEX)
     {
     case GGO_BITMAP:
-        load_flags |= FT_LOAD_TARGET_MONO;
+        load_flags |= FT_LOAD_TARGET_LIGHT;
         break;
     case GGO_GRAY2_BITMAP:
     case GGO_GRAY4_BITMAP:
     case GGO_GRAY8_BITMAP:
     case WINE_GGO_GRAY16_BITMAP:
-        load_flags |= FT_LOAD_TARGET_NORMAL;
+        load_flags |= FT_LOAD_TARGET_MONO;
         break;
     case WINE_GGO_HRGB_BITMAP:
     case WINE_GGO_HBGR_BITMAP:
