diff --git a/programs/explorer/explorer.c b/programs/explorer/explorer.c
index 1b900d0..a6efc3c 100644
--- a/programs/explorer/explorer.c
+++ b/programs/explorer/explorer.c
@@ -44,8 +44,8 @@ WINE_DEFAULT_DEBUG_CHANNEL(explorer);
 #define NAV_TOOLBAR_HEIGHT 30
 #define PATHBOX_HEIGHT 24
 
-#define DEFAULT_WIDTH 640
-#define DEFAULT_HEIGHT 480
+#define DEFAULT_WIDTH 800
+#define DEFAULT_HEIGHT 600
 
 
 static const WCHAR EXPLORER_CLASS[] = {'W','I','N','E','_','E','X','P','L','O','R','E','R','\0'};
