diff --git a/include/wine/wine_common_ver.rc b/include/wine/wine_common_ver.rc
index d8aeaf8..e3a6927 100644
--- a/include/wine/wine_common_ver.rc
+++ b/include/wine/wine_common_ver.rc
@@ -100,7 +100,7 @@ FILESUBTYPE    WINE_FILESUBTYPE
 {
     BLOCK "StringFileInfo"
     {
-	BLOCK "040904E4" /* LANG_ENGLISH/SUBLANG_DEFAULT, CP 1252 */
+	BLOCK "041103A4" /* LANG_JAPANESE/SUBLANG_DEFAULT, CP 932 */
 	{
 	    VALUE "CompanyName", "Microsoft Corporation"  /* GameGuard depends on this */
 	    VALUE "FileDescription", WINE_FILEDESCRIPTION_STR
@@ -115,6 +115,6 @@ FILESUBTYPE    WINE_FILESUBTYPE
     }
     BLOCK "VarFileInfo"
     {
-	VALUE "Translation", 0x0409, 0x04E4 /* LANG_ENGLISH/SUBLANG_DEFAULT, CP 1252 */
+	VALUE "Translation", 0x0411, 0x03A4 /* LANG_JAPANESE/SUBLANG_DEFAULT, CP 932 */
     }
 }
