diff --git a/dlls/wined3d/wined3d_private.h b/dlls/wined3d/wined3d_private.h
index b1c7089..8ad6e87 100644
--- a/dlls/wined3d/wined3d_private.h
+++ b/dlls/wined3d/wined3d_private.h
@@ -1455,7 +1455,7 @@ enum wined3d_pci_device
     CARD_NVIDIA_GEFORCE_GTX275      = 0x05e6,
     CARD_NVIDIA_GEFORCE_GTX280      = 0x05e1,
     CARD_NVIDIA_GEFORCE_315M        = 0x0a7a,
-    CARD_NVIDIA_GEFORCE_320M        = 0x08a3,
+    CARD_NVIDIA_GEFORCE_320M        = 0x08a2, /* MacbookAir3,1 (NVIDIA GeForce 320M) */
     CARD_NVIDIA_GEFORCE_GT320M      = 0x0a2d,
     CARD_NVIDIA_GEFORCE_GT325M      = 0x0a35,
     CARD_NVIDIA_GEFORCE_GT330       = 0x0ca0,
