diff --git a/include/wine/wine_common_ver.rc b/include/wine/wine_common_ver.rc
index 39a17d629b..1610159ca7 100644
--- a/include/wine/wine_common_ver.rc
+++ b/include/wine/wine_common_ver.rc
@@ -122,8 +122,8 @@ FILESUBTYPE    WINE_FILESUBTYPE
 {
     BLOCK "StringFileInfo"
     {
-        /* LANG_ENGLISH/SUBLANG_DEFAULT, WINE_CODEPAGE */
-	BLOCK "0409" WINE_CODEPAGE_STR
+        /* LANG_JAPANESE/SUBLANG_DEFAULT, WINE_CODEPAGE */
+	BLOCK "0411" WINE_CODEPAGE_STR
 	{
 	    VALUE "CompanyName", "Microsoft Corporation" /* GameGuard depends on this */
 	    VALUE "FileDescription", WINE_FILEDESCRIPTION_STR
@@ -138,7 +138,7 @@ FILESUBTYPE    WINE_FILESUBTYPE
     }
     BLOCK "VarFileInfo"
     {
-        /* LANG_ENGLISH/SUBLANG_DEFAULT, WINE_CODEPAGE */
-	VALUE "Translation", 0x0409, WINE_CODEPAGE_HEX
+        /* LANG_JAPANESE/SUBLANG_DEFAULT, WINE_CODEPAGE */
+	VALUE "Translation", 0x0411, WINE_CODEPAGE_HEX
     }
 }
