diff --git a/loader/wine_info.plist.in b/loader/wine_info.plist.in
index 6495b8f90f..fa5871f872 100644
--- a/loader/wine_info.plist.in
+++ b/loader/wine_info.plist.in
@@ -22,6 +22,8 @@
     <string>????</string>
     <key>CFBundleVersion</key>
     <string>@PACKAGE_VERSION@</string>
+    <key>NSHighResolutionCapable</key>
+    <true/>
     <key>NSPrincipalClass</key>
     <string>WineApplication</string>
     <key>LSUIElement</key>
